openfaas/faasd
# Update provider to v0.19

Signed-off-by: Nitishkumar Singh <nitishkumarsingh71@gmail.com>



## Description


## Motivation and Context


- [x] I have raised an issue to propose this change **this is required**
closes #289 

## How Has This Been Tested?



1. make local build after updating the provider version
2. add below changes in `prometheus.yml`. 
```
  - job_name: 'provider'
    static_configs:
    - targets: ['faasd-provider:8081']
```
 As faasd adds CNI gateway URL into **hosts** file, while creating container for services defined into `docker-compose.yaml`. Check the code [here](https://github.com/openfaas/faasd/blob/master/pkg/supervisor.go#L95)
3. Verify metrics into prometheus. if on multipass then replace line `- "127.0.0.1:9090:9090"` in ports section with `"9090:9090"`

![Screenshot from 2022-07-07 08-51-13](https://user-images.githubusercontent.com/11610230/177684925-c165c099-335d-4bba-8fb3-cec4ac0019ae.png)

## Types of changes


## Checklist:

Commits:

- [x] My commit message has a body and describe how this was tested and why it is required.
- [x] I have signed-off my commits with `git commit -s` for the Developer Certificate of Origin (DCO)

Code:


Docs:
